ci-operator/config/openshift/logging-view-plugin/openshift-logging-view-plugin-main.yaml

```diff
@@ -0,0 +1,29 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_literal: |
+    FROM cypress/included:3.8.0
+  to: plugin-test
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
we should add a test once CLO merges the PR which cross checks the install against CLO code